poker-pals/src/routes.js

```diff
@@ -4,6 +4,7 @@ import profileComponent from './components/Profile.vue';
 import reportsComponent from './components/Reports.vue';
 import tablesComponent from './components/Tables.vue';
 import exampleComponent from './components/HelloWorld.vue';
+import bannedComponent from './components/Banned.vue';
 
 export default [
     {path: '/', redirect: { name: 'Login'}},
```

Comment:
What will happen if you try to manually change the url from, say, `localhost:8080/#/login` to `localhost:8080/#/invalid`?

Reply:
maybe some else might want to give this whole single page navigation a shot